Trying to run psalm with wp-taint enabled produces an error.

Hi Tuncay, 

I'm currently running psalm & wp-taint as part of a Github Action for a WordPress setup.

Here's relevant workflow commands: 
```

      - name: Enable Psalm WP Taint Plugin
        run: |
          ./vendor/bin/psalm-plugin enable tuncay/psalm-wp-taint
        shell: bash
        
      - name: Show Psalm Plugins
        run: |
          ./vendor/bin/psalm-plugin show

      - name: Run Psalm with Taint Analysis
        run: |
          ./vendor/bin/psalm --taint-analysis --report=results.sarif --dump-taint-graph=taints.dot

```
Here's my composer.json

`{
    "require-dev": {
        "vimeo/psalm": "^5.25"
    },
    "require": {
        "tuncay/psalm-wp-taint": "dev-main"
    }
}`

When Github Action runs the following error is produced, if i disable the plugin psalm works as expected.

```
Uncaught UnexpectedValueException: Plugin is not a known class in /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Config.php:1762
Stack trace:
0 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Config.php(1643): Psalm\Config->loadPlugin()
1 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/ProjectAnalyzer.php(244): Psalm\Config->processPluginFileExtensions()
2 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Internal/Cli/Psalm.php(342): Psalm\Internal\Analyzer\ProjectAnalyzer->__construct()
3 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/psalm(9): Psalm\Internal\Cli\Psalm::run()
4 /home/runner/work/my-github-action/my-github-action/vendor/bin/psalm(119): include('...')
5 {main}

Next Psalm\Exception\ConfigException: Failed to load plugin Plugin in /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Config.php:1772
Stack trace:
0 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Config.php(1643): Psalm\Config->loadPlugin()
1 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/ProjectAnalyzer.php(244): Psalm\Config->processPluginFileExtensions()
2 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/src/Psalm/Internal/Cli/Psalm.php(342): Psalm\Internal\Analyzer\ProjectAnalyzer->__construct()
3 /home/runner/work/my-github-action/my-github-action/vendor/vimeo/psalm/psalm(9): Psalm\Internal\Cli\Psalm::run()
4 /home/runner/work/my-github-action/my-github-action/vendor/bin/psalm(119): include('...')
5 {main}
(Psalm 5.25.0@01a8eb06b9e9cc6cfb6a320bf9fb[14](https://github.com/jrgdiaz/my-github-action/actions/runs/10650692314/job/29522362259#step:7:15)331919d505 crashed due to an uncaught Throwable)
Error: Process completed with exit code 1.
```

What could be the issue? Thanks!

--- a/.gitignore
+++ b/.gitignore
@@ -4,6 +4,9 @@ composer.phar
 psalm.xml
 .phpunit.cache/
 coverage/
+add-actions-map.json
+psalm-report.xml
+psalm.xml
 
 # Commit your application's lock file https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control
 # You may choose to ignore a library lock file http://getcomposer.org/doc/02-libraries.md#lock-file

--- a/.gitignore
+++ b/.gitignore
@@ -6,7 +6,6 @@ psalm.xml
 coverage/
 add-actions-map.json
 psalm-report.xml
-psalm.xml
 
 # Commit your application's lock file https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control
 # You may choose to ignore a library lock file http://getcomposer.org/doc/02-libraries.md#lock-file

--- a/composer.json
+++ b/composer.json
@@ -19,7 +19,7 @@
   ],
   "extra": {
     "psalm": {
-      "pluginClass": "Plugin"
+      "pluginClass": "Tuncay\\PsalmWpTaint\\Plugin"
     }
   },
   "require": {

--- a/composer.json
+++ b/composer.json
@@ -1,6 +1,6 @@
 {
   "name": "tuncay/psalm-wp-taint",
-  "version": "0.0.1-dev",
+  "version": "0.0.2",
   "description": "A psalm plugin that adds stubs for psalm to define custom taint sinks for wordpress projects",
   "type": "psalm-plugin",
   "license": "proprietary",

